src/SidebarSectionItem/SidebarSectionItem.st.css

```
@@ -59,85 +65,73 @@
   background-color: value(selectedBackgroundColor)
 }
 
-.root:selected .text:size(small) {
-  color: value(B10);
+.root:skin(dark):selected .text {
+  color: value(selectedTextColor);
 }
 
 .root:skin(light):selected {
   background-color: value(selectedLightBackgroundColor)
 }
 
+.root:skin(light):selected .text {
+  color: value(selectedLightTextColor);
+}
+
 /* Hovered */
 
 .root:hover, .root:hover:selected:not(:disabled) {
   background-color: value(hoveredBackgroundColor);
   cursor: pointer;
 }
 
-.root:hover:not(:disabled) .text:size(small), .root:hover:selected:not(:disabled) .text:size(small) {
-  color: value(B30);
-}
-
 .root:skin(light):hover, .root:skin(light):hover:selected:not(:disabled) {
   background-color: value(hoveredLightBackgroundColor);
 }
 
-.root:skin(light):hover:not(:disabled) .text:size(small), .root:skin(light):hover:selected:not(:disabled) .text:size(small) {
-  color: value(hoveredLightTextColor);
-}
-
 /* Disabled */
 
 .root:disabled {
   background-color: value(disabledBackgroundColor);
   cursor: default;
 }
 
+.root:disabled .text {
+  color: value(disabledTextColor);
+}
 
 .root:skin(light):disabled {
   background-color: value(disabledLightBackgroundColor);
 }
 
+.root:skin(light):disabled .text {
+  color: value(disabledLightTextColor);
+}
+
 /* Others */
 
 .root:focus {
   -st-mixin: focus-box;
 }
-.root:drillable {
-  padding-right: 42px;
-}
-
-.root:drillable:hover, .root:drillable:alwaysDisplayChevron {
-  padding: value(paddingWithSuffix);
-}
 
 .root:drillable .chevron {
-  position: absolute;
+  color: value(D80);
   opacity: 0;
 }
 .root:drillable:alwaysDisplayChevron .chevron {
-  position: static;
-  opacity: value(chevronPartialOpacity);
+  opacity: 0.3;
```

@banuni This is supposed to be `0.6`, isn't it?
```suggestion
  opacity: 0.6;
```

---

Where do you see this in the spec? It was 0.3 before so I left it as it. Also the spec has an example for back button that uses 0.3.

---

@banuni insisted on `0.6` - it would be better to ask him

---

Okay. Nadav has confirmed that it should be 0.6.